presto-parquet/src/main/java/io/prestosql/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -316,6 +322,9 @@ public static Domain getDomain(Type type, DictionaryDescriptor dictionaryDescrip
         if (type.equals(DOUBLE) && columnDescriptor.getPrimitiveType().getPrimitiveTypeName() == PrimitiveTypeName.DOUBLE) {
             List<Domain> domains = new ArrayList<>();
             for (int i = 0; i < dictionarySize; i++) {
+                if (Double.isNaN(dictionary.decodeToDouble(i))) {
```

Comment:
extract `dictionary.decodeToDouble(i)` as variable

(shouldn't matter in practice but still)